test/SlowTests/Server/Documents/AI/GenAi/GenAiBasics.cs

```diff
@@ -0,0 +1,533 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using FastTests;
+using Newtonsoft.Json;
+using Raven.Client;
+using Raven.Client.Documents.Operations.AI;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Server.Documents;
+using Raven.Server.Documents.ETL.Providers.AI.GenAi;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Server.Documents.AI.GenAi;
+
+public class GenAiBasics(ITestOutputHelper output) : RavenTestBase(output)
```

Comment:
Please add a test for export / import (backup / restore) of a database with Gen AI task defined.

I didn't see any changes in the Smuggler code so I believe this is still TODO.

Reply:
done
see `GenAiBackupRestore.cs`